<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Application</class>
 <widget class="QMainWindow" name="Application">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CryptAnalyst</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="plaintextLabel">
           <property name="text">
            <string>Plaintext</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="plaintextEdit"/>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="1">
            <widget class="QComboBox" name="keyType">
             <item>
              <property name="text">
               <string>&lt;Key&gt;</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Random</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Custom...</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="cipherType">
             <item>
              <property name="text">
               <string>&lt;Cipher Type&gt;</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Affine</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Aristocrat</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Atbash</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Caesar</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Patristocrat</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="encodeButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Encode</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="ciphertypeLabel">
             <property name="text">
              <string>Cipher Type</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="keyLabel">
             <property name="text">
              <string>Character Key</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="separator">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="ciphertextLabel">
         <property name="text">
          <string>Ciphertext</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="ciphertextEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_Plaintext"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAnalysis">
    <property name="title">
     <string>Analysis</string>
    </property>
    <addaction name="actionLetter_Frequency"/>
    <addaction name="actionWord_Patterns"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionCheck_for_Updates"/>
    <addaction name="actionAbout_CryptAnalyst"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAnalysis"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLetter_Frequency">
   <property name="text">
    <string>Letter Frequency</string>
   </property>
  </action>
  <action name="actionWord_Patterns">
   <property name="text">
    <string>Word Patterns</string>
   </property>
  </action>
  <action name="actionAbout_CryptAnalyst">
   <property name="text">
    <string>About CryptAnalyst</string>
   </property>
  </action>
  <action name="actionImport_Plaintext">
   <property name="text">
    <string>Import Plaintext</string>
   </property>
  </action>
  <action name="actionCheck_for_Updates">
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>Application</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
